styles/myanimelist/catppuccin.user.css

```diff
@@ -0,0 +1,99 @@
+/* ==UserStyle==
+@name MyAnimeList Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/myanimelist
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/myanimelist
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/myanimelist/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Amyanimelist
+@description Soothing pastel theme for MyAnimeList
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('myanimelist.net') {
+    /* Exclude animelist and mangalist pages */
+    @-moz-document regexp("^(?!.*myanimelist\\.net\\/(animelist|mangalist)\\/).*$") {
```

Comment:
Why are we not using this regex instead of the `domain(...)` above?

Reply:
oh good point